doc/adr/0003-publisher-not-using-react.md

```diff
@@ -7,11 +7,18 @@ Date: 2019-10-01
 Accepted
 
 ## Context
-Libero Publisher needs a read-optimised, highly performant, progressively enhanced front end.
+Investigation into stronger alignment of technologies, practices and processes across the Libero Suite of products has resulted in time-boxed discovery sessions to find the advantages and consequences of various different aspects.
+One such area was front-end technologies. Other areas looked at aligning around JavaScript technologies and removing PHP from Libero Publisher.
 
-## [Decision](https://docs.google.com/document/d/1FkdvBjSb1BH1fhgujtppunvL_NX5U3xVuxv7L0fV010/edit#heading=h.oip2sudw7pju)
+Libero Reviewer uses React and JavaScript so the hypothesis was that aligning Libero Publisher around those technologies would improve the developer experience and encourage more reuse between the products. However, Libero Publisher needs a read-optimised, highly performant, progressively enhanced front end. This is a stronger need for Publisher than for Reviewer or Producer.
```

Comment:
My take on "This is a stronger need for Publisher than for Reviewer or Producer" would be closer to "This need has been deprioritised in Producer and Reviewer, but Publisher has a strong need for this."